Get started update (data mode) (#3955)
timescale/docs: _partials/_cloud-connect-service.md

<Procedure>

1. **Check your $SERVICE_SHORT is running correctly**

   In [$CONSOLE][services-portal], check that your $SERVICE_SHORT is marked as `Running`.

   ![Check $SERVICE_SHORT is running](https://assets.timescale.com/docs/images/console-services-view.png)

1. **Connect to your $SERVICE_SHORT**

   Connect using SQL editor in $CONSOLE or with psql in the command line:

   <Tabs label="Connect to your Timescale Cloud service">
   
   <Tab title="SQL editor in Console">

   <Procedure>

   1. In $CONSOLE, select your $SERVICE_SHORT.
   
   1. Click `SQL editor`. 

> 

Reviewer: Hey @atovpeko would you kindaly add a step where the user runs a basic query?

Author: Done!

<Procedure>

1. **Check your $SERVICE_SHORT is running correctly**

   In [$CONSOLE][services-portal], check that your $SERVICE_SHORT is marked as `Running`.

   ![Check $SERVICE_SHORT is running](https://assets.timescale.com/docs/images/console-services-view.png)

1. **Connect to your $SERVICE_SHORT**

   Connect using SQL editor in $CONSOLE or with psql in the command line:

   <Tabs label="Connect to your Timescale Cloud service">
   
   <Tab title="SQL editor in Console">

   <Procedure>

   1. In $CONSOLE, select your $SERVICE_SHORT.
   
   1. Click `SQL editor`. 

      ![SQL editor](https://assets.timescale.com/docs/images/ops-view-sql-editor.png)

   1. Run a test query:

      ```sql
      SELECT CURRENT_DATE;
      ```

>       If this query gives you the current date, then you have successfully connected to your $SERVICE_SHORT.

Reviewer 2: ```suggestion
      This query returns the current date, you have successfully connected to your $SERVICE_SHORT.
```
Never show doubt.

   
   </Procedure>

   </Tab>
   
   <Tab title="psql on the command line">

   <Procedure>

   1. Install [psql][psql].

   1. Run the following command in the terminal using the service URL from the config file you have saved during service creation:

      ```
      psql "<your-service-url>"
      ```

   1. Run a test query:

      ```sql
      SELECT CURRENT_DATE;
      ```

      If this query gives you the current date, then you have successfully connected to your $SERVICE_SHORT.

   </Procedure>

   </Tab>

   </Tabs>

> 

Reviewer 2: Can you move the "and that is it" to here please.

Author: Somehow if it's moved here, it does not show, so I moved it to right after the tabs

Reviewer 2: My bad, needs to be before the tab.
